zebra-consensus/Cargo.toml

```
@@ -15,9 +15,11 @@ bellman = "0.10.0"
 bls12_381 = "0.5.0"
 chrono = "0.4.19"
 displaydoc = "0.2.2"
+halo2 = "=0.1.0-beta.1"
 jubjub = "0.7.0"
 lazy_static = "1.4.0"
 once_cell = "1.8"
+orchard = "0.0"
```

Nit: depend on patch releases.

```suggestion
orchard = "0.0.0"
```